doc/releases/migration-guide-3.7.rst

```diff
@@ -177,6 +177,14 @@ LED Strip
 * The property ``in-gpios`` defined in :dtcompatible:`worldsemi,ws2812-gpio` has been
   renamed to ``gpios``.
 
+* The ``chain-length`` and ``color-mapping`` properties have been added to all LED strip bindings
+  and are now mandatory.
+
+* Added a new mandatory ``length`` function which returns the length (number of pixels) of an LED
```

Comment:
Since LED begins with a consonant sound, shouldn't it be "of a LED" instead ?

Reply:
It's actually a vowel sound: ... an el-ee-dee ...

Reply:
You are right ! For some reason I had the french sound in mind :)